### PR TITLE
container.fc: label /etc/crio/...

### PR DESCRIPTION
**Pull request by runcom**
@rhatdan PTAL

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
Ping @rhatdan

File: container.fc [hunk 1]
```diff
@@ -12,15 +12,14 @@
 /usr/bin/docker-novolume-plugin	--	gen_context(system_u:object_r:container_auth_exec_t,s0)
 /usr/sbin/crio.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/bin/crio.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
-/usr/bin/ocid.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
```
Comment (rhatdan) [inline container.fc]: I don't think we should remove these until we release cri-o.  So leave these in for now.

Comment (runcom) [inline container.fc]: brought these back

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**